modules/jaggery-apps/portal/gadgets/consent_management/js/gadget.js

```diff
@@ -503,7 +504,7 @@ function constructDate(expiry) {
     if (!checkValidDate(date)) {
         return getDefaultExpiry();
     }
-    var goodDate = date.getMonth() + "/" + date.getDate() + "/" + date.getFullYear();
+    var goodDate = date.getMonth() + 1 + "/" + date.getDate() + "/" + date.getFullYear();
```

Comment:
Don't we have a way to construct the date directly without appending each part ? date format ?

Reply:
@ayshsandu @thusithak Thoughts ?

Reply:
AFAIK there aren't any other straightforward way with vanilla js to achieve the same.
We could use some js library like [moment.js](https://momentjs.com/), which will be an overhead since we might not be doing alot of date format and manipulations.